index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="app-wrap">
+        <header class="app-header">
+            <div class="logo"><p class="text_logo">My Logo</p></div>
+            <nav class="nav-bar">
+                 <div class="nav">
+                    <img class="menu" src="Union.png" alt="menu">
+                </div>  
+            </nav>
+        </header>
+        <div class="content">
+            <h2 class="salute">I'm Alexandra. Glad to see you!</h2>
+            <p class ="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <p class="lets_connect">LET’S CONNECT</p>
+            <div class="social-links">
+                <a class="social" href="#"><img src="SocialMediaIcons\Twitter_icon.png" alt="Twitter_icon"></a>
+                <a class="social" href="#"><img src="SocialMediaIcons\Linkedin_icon.png" alt="Linkedin_icon"></a>
+                <a class="social" href="#"><img src="SocialMediaIcons\M_icon.png" alt="M"></a>
+                <a class="social" href="#"><img src="SocialMediaIcons\Github_icon.png" alt="Github_icon"></a>
+                <a class="social" href="#"><img src="SocialMediaIcons\Hand_icon.png" alt="Hand_icon"></a>
+            </div>

```

Comment:
- [ ] Well done using the `a` tag for the icons, but it is better if you use an unordered list `ul` to list `li` all the icons 📃

Reply:
isn't a tag for links?, Should I not use anchor?